flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointITCase.java

```diff
@@ -97,7 +97,7 @@
  * </ul>
  */
 @RunWith(Parameterized.class)
-@Category(FailsWithAdaptiveScheduler.class) // FLINK-21400
+@Category(FailsWithAdaptiveScheduler.class) // TODO: open new ticket
```

Comment:
This test currently just does not terminate, so something else appears to be amiss. I couldn't narrow it down because the test is incomprehensible.

Reply:
FLINK-21689

Reply:
TODO can be replaced with reference to `FLINK-21689`.